lib/Extension/Rpc/RpcExtension.php

```diff
@@ -61,6 +63,17 @@ public function load(ContainerBuilder $container): void
             return new LazyContainerHandlerRegistry($container, $handlers);
         });
 
+        if (class_exists(DebugExtension::class)) {
```

Comment:
I thnk it would eb fine without the class_exists?

Reply:
Yes, I removed it.